shopinvader_api_delivery_carrier/tests/common.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# Copyright 2023 ACSONE SA/NV.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import uuid
+
+from odoo.addons.extendable_fastapi.tests.common import FastAPITransactionCase
+from odoo.addons.shopinvader_api_cart.routers import cart_router
+
+
+class TestShopinvaderDeliveryCarrierCommon(FastAPITransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.partner = cls.env["res.partner"].create(
+            {"name": "FastAPI Delivery Carrier Demo"}
+        )
+        cls.user_with_rights = cls.env["res.users"].create(
```

Comment:
always set `no_reset_password` in ctx when creating users (or use new_test_user from odoo.tests.common)

Reply:
Done (using `new_test_user`)